conf/supportability.yaml

```diff
@@ -1,4 +1,4 @@
 supportability:
   content_hosts:
     rhel:
-      versions: [6, 7, 8, 9]
+      versions: [6, 7,'7_fips', 8, '8_fips', 9]
```

Comment:
all the tests that are now using regex matcher with not for example `@pytest.mark.rhel_ver_match('[^6].*')` will now run the test on the fips hosts as well,  but IMO this is not correct, 

The first solution in my mind is, can you create a separate list for fips hosts?

Reply:
Hm, so you suggest fips hosts should be run only when rpecifically requested, not when running with matcher or all hosts?

Reply:
I don't know much about fips, so you have better understanding, 
I do suggest maybe you can have different queue which will match fips

```
rhel:
  versions: [6,7,8,,9]
rhel_fips:
  versions: [7,8]
```
@rplevka  may have better ideas as he implemented it.

Reply:
For the time being, I'd include FIPS in all tests by default (opt-out)

Reply:
we could figure it out in here https://github.com/SatelliteQE/robottelo/blob/024d2df1b72b0425bce79a82487726b0dba47294/pytest_plugins/fixture_markers.py#L8 (note to self)   

Lets discuss on JFT later today